app/assets/javascripts/provider/admin/apiconfig/services/service_discovery.js

```diff
@@ -57,10 +57,15 @@ document.addEventListener("DOMContentLoaded", function(e) {
     opt.text = val;
     return selectElem.appendChild(opt);
   };
+  
+  function safe_fetch(url, init) {
+    const tempRequest = new Request(url, init);
```

Comment:
Same here in the tempRequest, is any specific reason why to use the `Request` interface of Fetch API and not using fetch directly? I feel I'm missing something.